pkgs/development/compilers/llvm/3.9/llvm.nix

```diff
@@ -39,7 +42,16 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" ] ++ stdenv.lib.optional enableSharedLibraries "lib";
 
-  buildInputs = [ perl groff cmake libxml2 python libffi ]
+  nativeBuildInputs = [
+    perl
+    cmake
+    python
+  ];
+
+  buildInputs = [
+    groff
+    libxml2
+    libffi ]
```

Comment:
Not to be a big nit, but `]` on the line below before `++` I think would be more idiomatic.